src/Storages/HDFS/StorageHDFS.cpp

```diff
@@ -461,6 +461,7 @@ class HDFSSink : public SinkToStorage
     {
         std::lock_guard lock(cancel_mutex);
         finalize();
+        finalize();
```

Comment:
Why?

Reply:
It was added while testing, it's not needed, just forgot to remove. Thanks for noticing!